sdk/openai/openai/review/openai-types.api.md

```diff
@@ -67,17 +45,13 @@ export interface AzureCosmosDBChatExtensionConfiguration extends AzureChatExtens
 
 // @public
 export interface AzureCosmosDBChatExtensionParameters {
-    allow_partial_result?: boolean;
     authentication?: OnYourDataAuthenticationOptions;
     container_name: string;
     database_name: string;
     embedding_dependency: OnYourDataVectorizationSource;
     fields_mapping: AzureCosmosDBFieldMappingOptions;
     in_scope?: boolean;
-    include_contexts?: OnYourDataContextProperty[];
     index_name: string;
```

Comment:
```suggestion
    index_name: string;
    role_information?: string;
```
This property is included in the GA spec in `inference.json` [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AzureOpenAI/inference/stable/2024-06-01/inference.json#L1696-L1700). It's marked as removed from Aug preview API in TypeSpec

Reply:
I think it is a mistake to include it, I am leaving it out for now until we get confirmation it is included in the GA. See the release notes in https://learn.microsoft.com/en-us/azure/ai-services/openai/api-version-deprecation#changes-between-2024-07-01-preview-and-2024-08-01-preview-api-specification